src/components/Global/TotalValue.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react'
+import { CountUp } from 'countup.js'
+
+import "./Global.css"
+
+const TotalValue = ({ value, text }) => {
```

Comment:
It would be nice for TotalValue to have his own folder with a css file so we could move it easily if needed. What do you think of it ?